### PR TITLE
KOQU & EGKB ILS

### PR DESCRIPTION
**Pull request by ewanfleming**
Had to make another PR, weird stuff happened.

### CODE REVIEW

File: Glideslope.json [hunk 1]
```diff
@@ -4073,13 +4073,13 @@
     "airportICAO": "EGKB",
     "identifier": "IBGH",
     "associatedRunwayNumber": "21",
-    "latitude": 51.33702087,
-    "longitude": 0.03503300,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,
```
Comment (ItsW1ll) [inline Glideslope.json]: Change this to  just 0.03805548 :)

File: Glideslope.json [hunk 2]
```diff
@@ -4073,13 +4073,13 @@
...
     "elevation": 598,
     "frequency": 109350,
     "receptionRange": 10,
     "bearing": 206.062,
```
Comment (ItsW1ll) [inline Glideslope.json]: Was there a bearing change? if so add it here

File: LOC.json [hunk 3]
```diff
@@ -4023,12 +4023,12 @@
     "airportICAO": "EGKB",
     "identifier": "IBGH",
     "associatedRunwayNumber": "21",
-    "latitude": 51.32336044,
-    "longitude": 0.02660600,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,
```
Comment (ItsW1ll) [inline LOC.json]: And here as well. Needs to be 0.03805548

File: Glideslope.json [hunk 4]
```diff
@@ -20089,13 +20089,13 @@
     "airportICAO": "KOQU",
     "identifier": "IOQU",
     "associatedRunwayNumber": "16",
-    "latitude": 41.60567856,
-    "longitude": -71.41841888,
-    "elevation": 6,
+    "latitude": 41.60775048,
+    "longitude": -071.42169088,
```
Comment (ItsW1ll) [inline Glideslope.json]: Remove the zero in front of the 71 so it reads -71.42169088

File: Glideslope.json [hunk 5]
```diff
@@ -20089,13 +20089,13 @@
...
+    "elevation": 18.0,
```
Comment (ItsW1ll) [inline Glideslope.json]: Remove the .0 travis doesn't like it

File: LOC.json [hunk 6]
```diff
@@ -1,4 +1,4 @@
-[
+§[
```
Comment (ItsW1ll) [inline LOC.json]: @Adrien-Z i'm not quite sure on this edit? 🤔

File: LOC.json [hunk 7]
```diff
@@ -4023,12 +4023,12 @@
...
     "associatedRunwayNumber": "21",
-    "latitude": 51.32336044,
-    "longitude": 0.02660600,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,
     "elevation": 598,
     "frequency": 109350,
     "receptionRange": 18,
-    "bearing": 205.647
+    "bearing": 205.65
```
Comment (ItsW1ll) [inline LOC.json]: add a zero onto the end

File: Glideslope.json [hunk 8]
```diff
@@ -20089,13 +20089,13 @@
...
     "frequency": 109500,
     "receptionRange": 10,
-    "bearing": 145.188,
-    "glideslope": 3.00145094
+    "bearing": 145.09,
+    "glideslope": 3.0014509
```
Comment (ItsW1ll) [inline Glideslope.json]: add a zero on the end of both the "bearing" and "glideslope"

**Comment by ewanfleming** [comment]
Great, will fix that stuff and recommit. Thanks

Sent from my iPhone

On 31 Jan 2018, at 16:10, Will <notifications@github.com<mailto:notifications@github.com>> wrote:


@ItsW1ll requested changes on this pull request.

Good work for your first ILS edit. Just a few things need changes. Noting too major

________________________________

In Glideslope.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165100058>:

> @@ -4073,13 +4073,13 @@
     "airportICAO": "EGKB",
     "identifier": "IBGH",
     "associatedRunwayNumber": "21",
-    "latitude": 51.33702087,
-    "longitude": 0.03503300,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,


Change this to just 0.03805548 :)

________________________________

In Glideslope.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165100230>:

> @@ -4073,13 +4073,13 @@
     "airportICAO": "EGKB",
     "identifier": "IBGH",
     "associatedRunwayNumber": "21",
-    "latitude": 51.33702087,
-    "longitude": 0.03503300,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,
     "elevation": 598,
     "frequency": 109350,
     "receptionRange": 10,
     "bearing": 206.062,


Was there a bearing change? if so add it here

________________________________

In LOC.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165100524>:

> @@ -4023,12 +4023,12 @@
     "airportICAO": "EGKB",
     "identifier": "IBGH",
     "associatedRunwayNumber": "21",
-    "latitude": 51.32336044,
-    "longitude": 0.02660600,
+    "latitude": 51.33824862,
+    "longitude": 000.03805548,


And here as well. Needs to be 0.03805548

________________________________

In Glideslope.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165100777>:

> @@ -20089,13 +20089,13 @@
     "airportICAO": "KOQU",
     "identifier": "IOQU",
     "associatedRunwayNumber": "16",
-    "latitude": 41.60567856,
-    "longitude": -71.41841888,
-    "elevation": 6,
+    "latitude": 41.60775048,
+    "longitude": -071.42169088,


Remove the zero in front of the 71 so it reads -71.42169088

________________________________

In Glideslope.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165100906>:

> @@ -20089,13 +20089,13 @@
     "airportICAO": "KOQU",
     "identifier": "IOQU",
     "associatedRunwayNumber": "16",
-    "latitude": 41.60567856,
-    "longitude": -71.41841888,
-    "elevation": 6,
+    "latitude": 41.60775048,
+    "longitude": -071.42169088,
+    "elevation": 18.0,


Remove the .0 travis doesn't like it

________________________________

In LOC.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165101341>:

> @@ -1,4 +1,4 @@
-[
+§[


@Adrien-Z<https://github.com/adrien-z> i'm not quite sure on this edit? 🤔

________________________________

In LOC.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165101502>:

>      "elevation": 598,
     "frequency": 109350,
     "receptionRange": 18,
-    "bearing": 205.647
+    "bearing": 205.65


add a zero onto the end

________________________________

In Glideslope.json<https://github.com/InfiniteFlightAirportEditing/Navigation/pull/765#discussion_r165101827>:

>      "frequency": 109500,
     "receptionRange": 10,
-    "bearing": 145.188,
-    "glideslope": 3.00145094
+    "bearing": 145.09,
+    "glideslope": 3.0014509


add a zero on the end of both the "bearing" and "glideslope"

**Comment by ItsW1ll** [comment]
When committing can you please only submit EGKB then ill check it and you can go on to add KOQU

**Comment by ewanfleming** [comment]
I will try. I’m slightly confused as I seem to be unable to choose which commits to make PRs for. I’ll have a go but may need a spot of help. Thanks

**Comment by ItsW1ll** [comment]
okay, so "fork" the latest version of the development branch

**Comment by ewanfleming** [comment]
Will be making another PR for each change.

**Pull request closed without merging**